<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wavepage</class>
 <widget class="QWidget" name="wavepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>294</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QLabel" name="label_Data">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="Wave_Screen">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="text">
         <string>截屏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Wave_Open">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="text">
         <string>打开文件夹</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="text">
         <string>波形测试</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="Serial_Wave_ON">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="text">
         <string>绘图显示</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Serial_Wave_Clean">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>27</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="text">
         <string>X轴</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Serial_Wave_XMode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>计数</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>时间</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Serial_Wave_Dot">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="text">
         <string>显示点</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Serial_Wave_New">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="text">
         <string>跟随最新数据</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Serial_Wave_Zoom">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="text">
         <string>X轴缩放</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Serial_Wave_ZoomAuto">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="text">
         <string>自适应</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Serial_Help">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="text">
         <string>?</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="verticalGroupBox">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="MyCustomPlot" name="qcpWidget" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyCustomPlot</class>
   <extends>QWidget</extends>
   <header>Wave/mycustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
